scripts/dac_setup.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+set -e
+
+usage="
+Usage: $(basename "$0") [-h] [-v VERSION]
+This script takes care of adding the CUE sources from Perses as external dependencies to your DaC repo.
+/!\ It must be executed at the root of your repo.
+
+Options:
+    -h  show this help text
+    -v  version of Perses (in the form of \"X.Y.Z\") from which to retrieve the dependencies
+
+Example: 
+$ ./dac_setup.sh -v 0.42.1"
```

Comment:
to ease the cross platform support, why not including that in the CLI ? 

like under the command `percli dac init --language=cue`

Reply:
Yes since we already planned to add such `dac` subcommand in the CLI for other stuff, I was thinking of something like this but for a next PR, but yes we can already do this move if you want